frontend/simple/main.js

```diff
@@ -41,7 +41,15 @@ async function loadLastUser () {
   /* eslint-disable no-new */
   new Vue({
     router: router,
-    components: {NavBar},
+    components: { NavBar },
+    computed: {
+      showSideBar () {
+        console.log('router:::', router.name)
+
+        // QUESTION: How to I hide the sidebar if the router is "home" ?
+        return true // router.name === 'home' // undefined...
```

Comment:
I think it would be best to let me and/or @hubudibu think about how to handle this part. For this PR if you could delete this it and open an issue and assign it to one or both of us it would be helpful.

Reply:
Done. https://github.com/okTurtles/group-income-simple/issues/437

Reply:
Actually I find it better the nav to be hidden until the user is logged. `this.$store.state.loggedIn`